exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -107,3 +116,42 @@ func NormalizeSpanName(tag string) string {
 func NormalizeSpanKind(kind pdata.SpanKind) string {
 	return strings.TrimPrefix(kind.String(), "SPAN_KIND_")
 }
+
+// ported from https://github.com/DataDog/datadog-agent/blob/eab0dde41fe3a069a65c33d82a81b1ef1cf6b3bc/pkg/trace/traceutil/normalize.go#L72
+// fallbackServiceNames is a cache of default service names to use
+// when the span's service is unset or invalid.
+var fallbackServiceNames sync.Map
```

Comment:
Instead of using `sync.Map` I would use `ttlmap` ([see here](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/internal/common@v0.17.0/ttlmap)). That way we avoid having the map growing indefinitely if we have high load/a bug.

Reply:
on second thought i'd scrapped all this fallback stuff for now since we aren't taking into account lang anyway, the fallback can be hardcoded